### PR TITLE
Add 2.6.0 release notes

### PR DESCRIPTION
**Pull request by VijayanB**
### Description
Add release notes

### Issues Resolved

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by junqiu-lei — APPROVED** [review]

File: release-notes/opensearch-dashboards-maps.release-notes-2.6.0.0.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+## Version 2.6.0.0 Release Notes
+Compatible with OpenSearch and OpenSearch Dashboards Version 2.6.0
+
+### Features
+* Add map as embeddable to dashboard ([#231](https://github.com/opensearch-project/dashboards-maps/pull/231))
+* Add maps saved object for sample datasets ([#240](https://github.com/opensearch-project/dashboards-maps/pull/240))
+
+### Infrastructure
+* [Cypress fix] Wait map saved before open maps listing ([#218](https://github.com/opensearch-project/dashboards-maps/pull/218))
```
Comment (jmazanec15) [inline release-notes/opensearch-dashboards-maps.release-notes-2.6.0.0.md:9]: Remove brackets

File: release-notes/opensearch-dashboards-maps.release-notes-2.6.0.0.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+### Refactoring
+* Refactor add layer operations ([#222](https://github.com/opensearch-project/dashboards-maps/pull/222))
+* Refactor layer operations ([#224](https://github.com/opensearch-project/dashboards-maps/pull/224))
+* Refactor layer properties as own interface ([#225](https://github.com/opensearch-project/dashboards-maps/pull/225))
+
+### Enhancements
+* Fix popup display while zoomed out ([#226](https://github.com/opensearch-project/dashboards-maps/pull/226))
```
Comment (jmazanec15) [inline release-notes/opensearch-dashboards-maps.release-notes-2.6.0.0.md:17]: Is this a bug?

**Review by naveentatikonda — APPROVED** [review]

File: release-notes/opensearch-dashboards-maps.release-notes-2.6.0.0.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+* Limit max number of layers ([#216](https://github.com/opensearch-project/dashboards-maps/pull/216))
+* Add close button to tooltip hover ([#263](https://github.com/opensearch-project/dashboards-maps/pull/263))
+* Add scroll bar when more layers added ([#254](https://github.com/opensearch-project/dashboards-maps/pull/254))
+* Align items in add new layer modal ([#256](https://github.com/opensearch-project/dashboards-maps/pull/256))
```
Comment (jmazanec15) [inline release-notes/opensearch-dashboards-maps.release-notes-2.6.0.0.md:21]: Whats a modal?

Comment (junqiu-lei) [reply release-notes/opensearch-dashboards-maps.release-notes-2.6.0.0.md:21]: A UI component. https://oui.1291fa08092a4a2dfd6d43f11d7bbcf3f22bf569.opensearch.org/latest/#/layout/modal

**Pull request merged**